Libraries/Animated/nodes/AnimatedStyle.js

```diff
@@ -51,7 +62,10 @@ export default class AnimatedStyle extends AnimatedWithChildren {
   }
 
   __getValue(): Object {
-    return this._walkStyleAndGetValues(this._style);
+    return flattenStyle([
```

Comment:
You can't flatten styles on web for the reasons mentioned above

Reply:
I guess changing this return value to an array would cause a breaking change right?

Reply:
🤷

Reply:
Thoughts on this @javache?

Reply:
Sounds good @necolas, I've just updated this to return an array and also updated the unit tests to reflect this change

Reply:
What we could do is put this behind a `Platform.OS === 'web'` guard, to minimize the chances of unexpected consequences at this stage and have an easier time getting review approved. Maybe all we change for native is return the flattened object wrapped in an array, for consistency. We could probably include web-only unit tests to preserve the behavior we need for web.

Reply:
Makes sense, I've added the `Platform.OS === 'web'` along with web-only unit tests